### ISSUE TITLE
monitor val_loss in tensorboard

### ISSUE THREAD

**Issue by liminghuiv** [fizyr/keras-retinanet#952]
In keras_retinanet/bin/train.py, line 196~205, the train process adds train loss to the callbacks, and we can use tensorflow tensorboard to monitor the train loss, it will be better to monitor val_loss at the same time. But when I added: 
# https://www.programcreek.com/python/example/104413/keras.callbacks.ReduceLROnPlateau
    callbacks.append(keras.callbacks.ReduceLROnPlateau(
        monitor='val_loss',
        factor=0.1,
        patience=2,
        verbose=1,
        mode='auto',
        min_delta=0.0001,
        cooldown=0,
        min_lr=0
    ))
The val_loss does not show up in the tensorboard. Anybody has idea on what's wrong?

Thanks,

Liming

**Comment by hgaiser** [comment]
The callback you added monitors a value, in your case `val_loss`, and if it doesn't decrease for a while anymore between epochs then it lowers the learning rate. Did you pass the argument `--compute-val-loss`?

**Comment by liminghuiv** [comment]
Hi @hgaiser, thanks. I did not use the argument --compute-val-loss. And I searched the whole keras-retinanet repo. I did not find compute-val-loss.
Thanks and best regards,

Liming

**Comment by hgaiser** [comment]
Alright, I'll close this issue then, since there is no issue reported for our code.

**Comment by aginpatrick** [comment]
I get the same kind of "error". I don't know how to record val_loss. I set compute-val-loss to true but it does not seem to make a difference.  Can someone show where in the code we record the val_loss. I searched all the repo code but did not find anything.
